src/db/controller/impl/level.ts

```diff
@@ -23,9 +24,12 @@ export class LevelDbController extends EventEmitter implements IDatabaseControll
 
   private opts: LevelDBOptions;
 
-  public constructor(opts: LevelDBOptions) {
+  private logger: WinstonLogger;
+
+  public constructor(opts: LevelDBOptions, logger: WinstonLogger) {
```

Comment:
Could you wrap the second argument in an object? We've been currently observing a pattern in our constructors of arg1 = Options, arg2 = dependent modules.
eg:
`   public constructor(opts: LevelDBOptions, {logger: WinstonLogger}) { `

Reply:
Style guide alert.

Reply:
Implemented